engine/src/main/java/org/terasology/engine/entitySystem/sectors/SectorSimulationSystem.java

```diff
@@ -43,6 +41,8 @@
  * It also sends {@link OnChunkLoaded} and {@link BeforeChunkUnload} events to the entities, whenever the status of a
  * watched chunk changes. These should be captured by filtering only to entities with a
  * {@link SectorSimulationComponent}, to avoid capturing the event sent to the world entity.
+ *
+ * @see SectorUtil#getWatchedChunks(EntityRef) for the definition of wathed chunks.
```

Comment:
```suggestion
 * @see SectorUtil#getWatchedChunks(EntityRef) for the definition of watched chunks.
```

Since you're moving this line anyway, maybe fix the typo here too?

Reply:
done.